migrations/20210914160014_group_extensions.js

```diff
@@ -0,0 +1,35 @@
+exports.up = async function(knex) {
+    await knex.schema.createTable('extensions', table => {
+        table.increments().primary()
+        table.text('type')
+        table.timestamp('created_at')
+        table.timestamp('updated_at')
+    })
+
+    await knex.schema.createTable('group_extensions', table => {
+        table.increments().primary()
+        table.bigInteger('group_id').references('id').inTable('groups').notNullable()
+        table.bigInteger('extension_id').references('id').inTable('extensions').notNullable()
+        table.jsonb('data')
+        table.boolean('active').defaultsTo(true)
+        table.index('group_id')
+        table.timestamp('created_at')
+        table.timestamp('updated_at')
+    })
+
+
+    await knex.schema.alterTable('groups', async (table) => {
+      table.text('type').defaultsTo('group')
```

Comment:
this is a good question, do we always have a type? a default of 'group' seems kind of redundant and unecessary to me, i think we leave blank or null by default?